tools/build_config/build.ps1

```diff
@@ -188,6 +192,43 @@ function Invoke-Package {
   }
 }
 
+function Invoke-Build-Docs {
+  (Get-Content "./build/CPackConfig.cmake" | Where-Object {$_ -match 'CPACK_PACKAGE_FILE_NAME'}) -match '.*"Horace-([^"]+)".*'
+  $build_id = $Matches[1]
+  (Get-Content ./documentation/user_docs/docs/conf.py) | Foreach-Object {$_ -replace 'release = .*', "release = '${build_id}'"} | Set-Content  ./documentation/user_docs/docs/conf.py
+
+  ./documentation/user_docs/make.bat html
+  ./documentation/user_docs/make.bat html
```

Comment:
Is it necessary to run `make` twice? I see you're doing the same in Bash. If it is necessary, a comment explaining why would be good.

Reply:
Possibly a leftover habit from LaTeX paranoia. I'm not sure if it's necessary, I will do some testing.